packages/ubuntu_desktop_installer/lib/widgets/option_card.dart

```diff
@@ -67,15 +67,21 @@ class OptionCardState extends State<OptionCard> {
   @override
   Widget build(BuildContext context) {
     return Card(
+      color: Theme.of(context).cardColor,
       shape: RoundedRectangleBorder(
           side: BorderSide(
               color: widget.selected
-                  ? Theme.of(context).primaryColor.withOpacity(0.4)
-                  : Theme.of(context).colorScheme.onSurface.withAlpha(20),
-              width: 1),
-          borderRadius: BorderRadius.circular(4.0)),
-      elevation: widget.selected ? 0 : 1,
+                  ? Theme.of(context).primaryColor.withOpacity(0.5)
+                  : Theme.of(context)
+                      .colorScheme
+                      .onSurface
+                      .withAlpha(hovered ? 60 : 20),
+              width: 2),
```

Comment:
is it intentional that option card has 2px border whereas buttons and lists have 1px? (and textfield has 1px in unfocused, 2px focused)

Reply:
Yes, it the 1px border is too thin imo
Especially for the hover and selected effects.
But let's ask @waynecrosby what he thinks, even if he has already raised his thumb maybe the border width was not obvious to see

Reply:
ok, thanks! it just raised my attention so I wanted to double-check

Reply:
Button bars also have a  2px border

Reply:
my initial impression was that the 2px border looks strangely blurry, but I think it's actually the translucency that tricks my eye

Reply:
sorry, i must have missed something, but i can't see any toggle button bars in the installer. :hushed: is it a widget theme in yaru?

Reply:
It's a material widget called toggle group. Not in the installer though

Reply:
@Feichtmeier I'm happy with the 2px border for selected & hover states and agree with you that 1px is too thin